examples/aditof-test/main.cpp

```diff
@@ -339,13 +347,31 @@ int _DisplayIR(cv::Mat *irMat, int *measuredDistance, int targetDistance,
         }
     }
 
+    /* Read the camera temperature */
+    std::vector<std::shared_ptr<aditof::TemperatureSensorInterface>>
+        tempSensors;
+    camera->getTemperatureSensors(tempSensors);
+    tempSensors[0]->read(*temperature);
+
+    /* Save measured results in a file */
     std::ofstream MyFile("MeasuredResults.txt");
-    MyFile << "Targed set at distance: " << targetDistance << std::endl;
-    MyFile << "Camera measured: " << *measuredDistance << std::endl;
-    MyFile << "Precision: " << *precision;
+    MyFile << "Targed set at distance: " << targetDistance;
+    MyFile << "\nCamera measured: " << *measuredDistance;
+    MyFile << "\nPrecision: " << *precision;
+    MyFile << "\nTemperature: " << *temperature;
     MyFile.close();
 
-    std::string eepromID = "DUMMYVAL"; //to be read from eeprom for each camera
+    /* Read the camera serial from EEPROM */
+    uint8_t eepromName[12];
+    std::vector<std::shared_ptr<aditof::StorageInterface>> eeprom;
+    camera->getEeproms(eeprom);
+    eeprom[0]->read(0x00010016, eepromName, 12);
```

Comment:
Check if eeprom vector is not empty. Also maybe 'eeproms' would be a better name.